hibernate-core/src/test/java/org/hibernate/orm/test/jpa/remove/RemoveAndOrderUpdateTest.java

```diff
@@ -38,6 +40,7 @@
 		}
 )
 @JiraKey("HHH-16923")
+@SkipForDialect(dialectClass = AltibaseDialect.class, reason = "Altibase does not support identity column")
```

Comment:
There is no explicit use of the `IDENTITY` strategy, so not sure why this needs skipping. We actually have a special annotation to make tests conditional on DB features, in case of identity columns that would be `@RequiresDialectFeature(feature = SupportsIdentityColumns.class)`. I'd prefer to use that annotation over skipping dialects explicitly, but regardless, it seems something must be wrong with your `Dialect` or was in the past, as `Dialect#getNativeIdentifierGeneratorStrategy` should report `sequence` since this dialect does not support identity columns.

Reply:
I'm gonna check this as you mentioned

Reply:
I think this skip was created before finalize dialect class. After deleting skips,  this test runs ok. Thanks.